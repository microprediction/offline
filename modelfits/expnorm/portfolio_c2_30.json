{"g1": 0.031704520482507326, "g2": 12.156429985023092, "logK": 0.8425460077828273, "loc": -0.026198973873653143, "logScale": -0.2695015102123964}