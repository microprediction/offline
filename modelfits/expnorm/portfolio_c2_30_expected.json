{"g1": 0.3862618732719801, "g2": 13.786755548766987, "logK": 0.9275430916201499, "loc": -0.11082621868416431, "logScale": -2.0062767767391496}